tensordict/tensordict.py

```diff
@@ -3741,9 +3770,13 @@ def _stack_onto_(
 
     def get(
         self,
-        key: str,
+        key: NESTED_KEY,
         default: Union[str, COMPATIBLE_TYPES] = "_no_default_",
     ) -> COMPATIBLE_TYPES:
+        # TODO: the stacking logic below works for nested keys, but the key in
+        # self.valid_keys check will fail and we'll return the default instead.
+        # For now we'll advise user that nested keys aren't supported, but it should be
+        # fairly easy to add support if we could add nested keys to valid_keys.
```

Comment:
Should we raise an error here and tell the user that nested keys aren't supported by `LazyStackedTensorDict` yet, or wait for nested keys to be added to `self.valid_keys`?

Reply:
We can go in two phases
(1) raise an issue or add a bullet point in #22 for bookkeeping, and add nested keys to `valid_keys` in a second PR
(2) do it all together

I feel (1) makes more sense, we're in no rush and it's a specific usage